src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -266,6 +266,18 @@ public abstract partial class TransportContext
         protected TransportContext() { }
         public abstract System.Security.Authentication.ExtendedProtection.ChannelBinding GetChannelBinding(System.Security.Authentication.ExtendedProtection.ChannelBindingKind kind);
     }
+
+    public static class HttpVersion
```

Comment:
I opened https://github.com/dotnet/standard/issues/91 to discuss if it should be static